packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -168,14 +134,20 @@ abstract contract ResourceMetering is Initializable {
         }
     }
 
+    /**
+     * @notice Virtual function that returns the resource config. Contracts that inherit this
+     *         contract must implement this function.
+     */
+    function _resourceConfig() internal virtual returns (SystemConfig.ResourceConfig memory);
+
     /**
      * @notice Sets initial resource parameter values. This function must either be called by the
      *         initializer function of an upgradeable child contract.
      */
     // solhint-disable-next-line func-name-mixedcase
     function __ResourceMetering_init() internal onlyInitializing {
         params = ResourceParams({
-            prevBaseFee: INITIAL_BASE_FEE,
+            prevBaseFee: 1 gwei,
```

Comment:
Hmmm, need to rethink this a little in the case that 1 gwei is > than the max or < than the min.

Reply:
The problem is that you cannot access other contracts in the constructor due to the way that the deploy system works, but even if its out of the range of the max or min it should be clamped in on the first run